build-tools/packages/build-cli/src/commands/generate/assertTags.ts

```diff
@@ -97,71 +159,120 @@ export class TagAssertsCommand extends PackageCommand<typeof TagAssertsCommand>
 		}
 	}
 
-	protected async processPackage<TPkg extends Package>(
+	// This should not be called due to processPackages being overridden instead.
+	protected override async processPackage<TPkg extends Package>(
 		pkg: TPkg,
 		kind: PackageKind,
 	): Promise<void> {
-		const tsconfigPath = await this.getTsConfigPath(pkg);
-		this.collectAssertData(tsconfigPath);
+		throw new Error("Method not implemented.");
 	}
 
-	public async run(): Promise<void> {
-		// Calls processPackage on all packages to collect assert data.
-		await super.run();
+	protected override async processPackages(packages: PackageWithKind[]): Promise<string[]> {
+		const errors: string[] = [];
 
-		// Tag asserts based on earlier collected data.
-		this.tagAsserts(true);
-	}
+		const collected: CollectedData = {
+			shortCodes: new Map<number, Node>(),
+			codeToMsgMap: new Map<string, string>(),
+			maxShortCode: -1,
+		};
+
+		const dataMap = new Map<PackageWithKind, PackageData>();
+		const config = cosmiconfig("assertTagging");
```

Comment:
I recommend being explicit about the formats supported.

```suggestion
		const config = cosmiconfig("assertTagging", {
	searchPlaces: [
		`${configName}.config.cjs`,
		`${configName}.config.mjs`,
		]
	});
```

Reply:
Given that we should document what formats are supported and how to use them (I have now added this), I think its simpler to just point the docs fo cosmic config's listing as it covers how directories are searched and such. Given that, I don't see a reason not to just support what they support by default.

The other option that seems reasonable to me would be to force exactly one format for simplicity.

Personally though, I think if we want to just support one format, it would probably be better to put that decision elsewhere in a central location that says what formats all our tool use for config. Right now I'm using the cosmic config package's defaults as that central location, but if we end up wrapping it (ex: to pair it with typebox schema to do config validation and nice errors), then that util could probably be opinionated about config formats (and we could have some centralized docs, maybe in the build-cli readme, for how to configure tools).

Overall though, I don't care very much about this detail as long as its documented and consistent.

Reply:
I have limited it to look for config.mjs, and updated the docs accordingly.